.github/workflows/ci.yml

```diff
@@ -27,6 +27,8 @@ jobs:
         run: make dev
       - name: test
         run: make test
+      - name: test (offline)
+        run: sudo unshare -n make test TEST_ARGS="--skip-online"
```

Comment:
What's with the use of `sudo` and `unshare`? I think this deserves a comment.

Reply:
Yep, thanks for calling this out -- `unshare` is used to "un-share" the default networking namespace to the target program, meaning that it can't make network connections. In turn, that means that we exercise the correctness of our `pytest.mark.online` markers, since any tests that are missing markers would fail.

I'll summarize that in a comment! I think I can also work around the need for `sudo`.

Reply:
[8cd6330](https://github.com/sigstore/sigstore-python/pull/180/commits/8cd633011f6a59e9b248be05aa98d5669b10b100)

Reply:
Oh I see. Thanks for that.